src/rqt_console/main.py

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env python3
```

Comment:
I don't think the shebang line is necessary anymore, though I guess it doesn't hurt.